lua/modules/MyReadingManga.lua

```diff
@@ -1,82 +1,82 @@
-function getinfo()
-  mangainfo.url=MaybeFillHost(module.RootURL, url)
-  if http.get(mangainfo.url) then
-    local x=TXQuery.Create(http.document)
-    mangainfo.title=x.xpathstring('//*[contains(@class,"entry-content")]/h2')
-    if mangainfo.title == '' then
-      mangainfo.title = Trim(x.XPathString('//*[contains(@class,"entry-content")]/p[starts-with(.,"Title")]/substring-after(.,":")'))
-    end
-    if mangainfo.title == '' then
-      mangainfo.title = x.XPathString('//*[contains(@class,"entry-content")]/p[1]/strong')
-    end
-    if mangainfo.title == '' then
-      mangainfo.title = x.XPathString('//h1[@class="entry-title"]')
-    end
-    mangainfo.authors=Trim(x.xpathstring('//*[contains(@class,"entry-content")]/p[starts-with(.,"Author")]/substring-after(.,":")'))
-    mangainfo.genres=x.xpathstring('//header[@class="entry-header"]/string-join(./p[position()>1]//a,", ")')
-    mangainfo.chapterlinks.add(mangainfo.url)
-    mangainfo.chapternames.add(mangainfo.title)
-    local v = x.xpath('//*[contains(@class,"entry-pagination")]/a')
-    for i = 1, v.count do
-      local v1 = v.get(i)
-      if string.match(v1.toString, '^Next') == nil then
-        mangainfo.chapterlinks.add(v1.getAttribute('href'));
-        mangainfo.chapternames.add(mangainfo.title .. ' - ' .. v1.toString);
-      end
-    end
-    if mangainfo.chapternames.count > 1 then
-      mangainfo.chapternames[0] = mangainfo.chapternames[0] .. ' - 1'
-    end
-    return no_error
-  else
-    return net_problem
-  end
+function GetInfo()
+	mangainfo.URL = MaybeFillHost(module.RootURL, url)
+	mangainfo.Website = "MyReadingManga"
+	
+	if not http.get(mangainfo.URL) then return net_problem end
+	
+	local x = TXQuery.Create(http.Document)
+	
+	mangainfo.Title = x.XPathString("//h1[@class='entry-title']")
+	mangainfo.Genres = x.XPathString('//header[@class="entry-header"]/string-join(./p[position()>1]//a,", ")')
+	
+	mangainfo.ChapterLinks.Add(mangainfo.url)
+	mangainfo.ChapterNames.Add("1")
```

Comment:
This makes the chapter name just called `1`

Reply:
Oh yeah, it's a wordpress site and chapters (if there are chapters) are basically just numbered so I put it as 1. I guess if it's a oneshot I should put the manga name as the chapter title?

Reply:
Yes, other modules put the manga name as chapter title if there's no chapters.